sim/resources/stompypro/robot_fixed.xml

```diff
@@ -0,0 +1,191 @@
+<mujoco model="robot">
```

Comment:
This is not the xml I pushed? Remove damping information from all joints. Keep only default.